src/lib.rs

```diff
@@ -90,6 +93,8 @@ pub trait HasContext {
     type TransformFeedback: Copy + Clone + Debug + Eq + Hash + Ord + PartialEq + PartialOrd;
     type UniformLocation: Clone + Debug;
 
+    fn get_supported_extensions(&self) -> HashSet<String>;
```

Comment:
nit: could we remove the `get_` prefix to follow the convention in https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter?

Reply:
done